openadapt/alembic/versions/113f15863a33_regenrate_with_browser_event_table.py

```diff
@@ -0,0 +1,43 @@
+"""regenrate with browser_event table
```

Comment:
Can you please:
- run `alembic downgrade -4` to revert the four existing alembic scripts
- create a single new migration script with an appropriate description, e.g. "create BrowserEvent table"
- remove the old existing alembic migration scripts and add the new one